### PR TITLE
Apollo variant variable for env-specific metrics reporting

### PR DESCRIPTION
**Pull request by ryanahamilton**
A pre-req for astronomer/issues#1144.

### CODE REVIEW

File: prod/cloud/app/config.yaml [hunk 1]
```diff
@@ -102,7 +104,7 @@ alertmanager:
           {{ range .Alerts -}}
             *Alert:* {{ .Annotations.summary }}{{ if .Labels.severity }} - `{{ .Labels.severity }}`{{ end }}
           *Description:* {{ .Annotations.description }}
-          
+
```
Comment (sjmiller609) [inline prod/cloud/app/config.yaml:107]:
```suggestion
          
```
just in case

File: staging/cloud/app/config.yaml [hunk 2]
```diff
@@ -31,6 +31,8 @@ astronomer:
         value: "true"
       - name: APOLLO_KEY
         value: !Secret apollo_key
+      - name: APOLLO_GRAPH_VARIANT
+        value: "stage"
```
Comment (sjmiller609) [inline staging/cloud/app/config.yaml:35]: do we want 'stage' or 'staging'?

Comment (sjmiller609) [reply staging/cloud/app/config.yaml:35]: just to match convention, I vote 'staging'

File: prod/cloud/app/config.yaml (hunk 1 above)
Comment (ryanahamilton) [reply prod/cloud/app/config.yaml]: That was my IDE stripping whitespace on-save. I'll revert.

File: staging/cloud/app/config.yaml (hunk 2 above)
Comment (ryanahamilton) [reply staging/cloud/app/config.yaml:35]: Sorry, I thought that was the convention—will update!

**Pull request merged**